app/models/agent.rb

```diff
@@ -121,6 +121,10 @@ def timeout_in = 14.days # Used by Devise's :timeoutable
     where("invitation_sent_at IS NULL OR invitation_accepted_at IS NOT NULL")
   }
   scope :active, -> { where(deleted_at: nil) }
+  scope :excluding_pending_invitation, lambda {
+    where(invitation_sent_at: nil) # permet de ne pas inclure les intervenants & les ProConnectés
+      .or(Agent.where.not(confirmed_at: nil)) # plus générique que invitation_sent_at, évite les faux négatifs
```

Comment:
Encore un nitpick ultime 

```suggestion
      .or(where.not(confirmed_at: nil)) # plus générique que invitation_sent_at, évite les faux négatifs
```